src/plugins/intel_npu/src/al/CMakeLists.txt

```diff
@@ -0,0 +1,44 @@
+# Copyright (C) 2018-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+if(ENABLE_LTO)
+    set(CMAKE_INTERPROCEDURAL_OPTIMIZATION_RELEASE ON)
+endif()
+
+set(TARGET_NAME npu_al)
```

Comment:
can we have it `openvino_npu_al` to avoid issues with names conflicts?

Reply:
Have changed to openvino_npu_al.